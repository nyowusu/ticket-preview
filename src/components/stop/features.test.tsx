import { shallow } from 'enzyme';
import React from 'react';
import StopFeatures from './features';

it('renders correctly', () => {
  // test data
  const defaultProps = {
    stop: {
      type: 'Feature',
      geometry: {
        type: 'Point',
        coordinates: [20, 20],
      },
      properties: {
        stop_id: 'sDi',
        stop_name: 'name',
      },
    },
    updates: 8,
  };

  // @ts-ignore
  expect(shallow(<StopFeatures {...defaultProps} />)).toMatchSnapshot();
});
