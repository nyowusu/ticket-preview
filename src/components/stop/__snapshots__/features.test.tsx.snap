// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgba(0,0,0,0)",
        "display": "flex",
        "left": 0,
        "marginBottom": 10,
        "padding": 5,
        "paddingHorizontal": 15,
        "position": "absolute",
        "width": 750,
        "zIndex": 3,
      },
      Object {
        "bottom": 83,
      },
    ]
  }
>
  <View
    style={
      Object {
        "backgroundColor": "rgb(256, 256, 256)",
        "borderBottomColor": "transparent",
        "borderRadius": 5,
        "borderTopColor": "transparent",
        "borderWidth": 0,
        "elevation": 5,
        "shadowColor": "rgb(0, 0, 0)",
        "shadowOffset": Object {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0.8,
        "shadowRadius": 5,
      }
    }
  >
    <StopFeature
      containerStyles={
        Array [
          Object {
            "alignItems": "center",
            "display": "flex",
            "paddingVertical": 10,
          },
          Object {
            "borderBottomColor": "rgb(204, 204, 204)",
            "borderBottomWidth": 1,
          },
        ]
      }
      key="[missing \\"en.stop.features.departures\\" translation]"
      name="[missing \\"en.stop.features.departures\\" translation]"
      onPress={[Function]}
    />
    <StopFeature
      containerStyles={
        Array [
          Object {
            "alignItems": "center",
            "display": "flex",
            "paddingVertical": 10,
          },
          Object {
            "borderBottomColor": "rgb(204, 204, 204)",
            "borderBottomWidth": 1,
          },
        ]
      }
      key="[missing \\"en.stop.features.getMeHere\\" translation]"
      name="[missing \\"en.stop.features.getMeHere\\" translation]"
      onPress={[Function]}
    />
    <StopFeature
      containerStyles={
        Object {
          "alignItems": "center",
          "display": "flex",
          "paddingVertical": 10,
        }
      }
      key="[missing \\"en.stop.features.trackBuses\\" translation]"
      name="[missing \\"en.stop.features.trackBuses\\" translation]"
      onPress={[Function]}
    />
  </View>
</View>
`;
